charts/external-es-proxy/templates/external-es-proxy-configmap.yaml

```diff
@@ -43,10 +43,10 @@ data:
           {{- include "external-es-proxy-nginx-location-common" . | indent 10 }}
         }
 
-        location = /_search {
+        location ~* /_search$ {
           # This combined with disabling explicit index searching downstream
           # prevents any deployment from being able to query any other indexes.
-          rewrite ^/(.*) /{{ include "logging.indexNamePrefix" . }}.$remote_user.*/$1 break;
+          rewrite /_search(.*) /{{ include "logging.indexNamePrefix" . }}.$remote_user.*/_search$1 break;
```

Comment:
Does this actually work? It seems like an anchored regex `_search$` would not match `_search(.*)` unless the contents of `(.*)` were `$`

Reply:
this is as per our existing design , i have validated them in test cluster.

Reply:
Are you sure it works?

<img width="1073" alt="Screenshot 2024-05-07 at 9 17 27 AM" src="https://github.com/astronomer/astronomer/assets/1323808/1f958d62-5b58-4dcc-a03f-90954013d5b7">

https://nginx.viraptor.info

Reply:
Oooh, it does work with `_search?foo=bar`. Is that what we're matching?